package.json

```diff
@@ -87,6 +87,8 @@
   },
   "license": "LGPL-2.1-only",
   "dependencies": {
+    "@storybook/test": "^8.0.0",
+    "D": "^1.0.0",
```

Comment:
kesako?

Reply:
This is me, doing `npm i D @storybook/test` instead of `npm i -D @storybook/test` 👎🏻